addons/mail/static/src/models/thread/thread.js

```diff
@@ -1268,12 +1274,13 @@ function factory(dependencies) {
         message_unread_counter: attr({
             default: 0,
         }),
+        /**
+         * All messages that this thread is linked to.
+         * Note that this field is automatically computed by inverse
+         * computed field.
```

Comment:
Good that there is a comment.
Maybe specify it should be considered readonly?

For curiosity, what does it do if we link/unlink something on this field?

Reply:
> Good that there is a comment.
> Maybe specify it should be considered readonly?

Good idea
> For curiosity, what does it do if we link/unlink something on this field?
It makes the change on it, but any invocation of its compute method will revert changes (due to logic inside this compute method).

Reply:
Not done? (adding readonly comment)